test/idl/all.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert');
+
+const idl = require('@webref/idl');
+
+describe('@webidl/idl module', () => {
+  it('listAll', async () => {
+    const files = await idl.listAll();
+    assert.strictEqual(Object.keys(files).length, 198);
```

Comment:
I'm not sure I understand the value of hardcoding that number?

Reply:
I just wanted to make sure something fails if due to some mishap there are no files at all found, because I think all the other tests would still pass. How about I just check that `length` isn't 9?